wordle-web/README.md

```diff
@@ -0,0 +1,75 @@
+# Nuxt 3 Minimal Starter
+
+Look at the [Nuxt 3 documentation](https://nuxt.com/docs/getting-started/introduction) to learn more.
+
+## Setup
+
+Make sure to install the dependencies:
+
+```bash
+# npm
+npm install
+
+# pnpm
+pnpm install
+
+# yarn
+yarn install
+
+# bun
+bun install
+```
+
+## Development Server
+
+Start the development server on `http://localhost:3000`:
```

Comment:
I believe the external app should be here in the read me, not forced local. For example, ours is https://lemon-flower-0c0aec71e.5.azurestaticapps.net/

Reply:
Yep, missed that. Thanks